board_view.md

```diff
@@ -0,0 +1,273 @@
+  }
+},
+# [...]
+```
+
+Note that this relationship will *not* be present on the response for `/space/:id` as having normalized template information on the `space` relationships is deprecated. The client should always use the provided `space-template` relationship on the `space` entitiy to pull template information.
+
+The new external relationship `workitemboards` is available at the endpoint `/spacetemplates/:id/workitemboards` and has the following structure: 
+
+```yaml
+{
+  "id": "000-000-002",
+  "attributes": {
+    "name": "Scenarios Board",
+    "description": "This is the default board config for the legacy template (Experiences).",
+    "contextType": "TypeLevelContext",  // this designates the type of the context
+    "context": "000-000-003",  // this designates the ID of the context, in this case the typegroup ID
+    "created-at": "0001-01-01T00:00:00Z",
+    "updated-at": "0001-01-01T00:00:00Z"
+  },
+  "relationships": {
+    "spaceTemplate": {
+      "data": {
+        "id": "000-000-004",
+        "type": "spacetemplates"
+      }
+    },
+    "columns": {
+      "data": [
+        {
+          "id": "000-000-005",
+          "title": "workitemboardcolumn",
+          "order": 0,  // the left-to-right order of the column in the view
+          "relationships": {
+            "onUpdateActions": {  // should not be used in the UI
+              "data": [
+                {
+                  "id": "000-000-007",
+                  "type": "onUpdateActions",
+                  "transRuleKey": "updateStateFromColumnMove",
+                  "transRuleArguments": { 
+                    "metastate": "mNew" 
+                  }
+                }
+              ]
+            }
+          }
+          "type": "boardcolumns"
+        }
+      ]
+    }
+  },
+  "type": "workitemboards"
+}
+```
+
+Note that clients should always use the `related` link from the `/spacetemplate` response to pull the column definitions.
+
+There might be zero or more `onUpdateActions` defined on a `boardcolumns` definition. The definition, including the `transRuleKey` and `transRuleArguments` values are only used on the WIT side and should not be used by the UI. They are provided for a future use case where the user might be allowed to update those values to change the board behaviour. `transRuleKey` describes the rule/action that is executed when a Work Item is moved *into* this column while `transRuleArguments` gives additional parameters for this action. In the current state of the implementation, this will execute a rule that updates the state of the Work Item moved into the column based on the meta-state given in `transRuleArguments`. In the future, there might be other rules/actions and different arguments. One example could be to create columns for labels. In this case, the rule/action needs to update the labels on the Work Item and the `transRuleArguments` may contain the label attached to the described column.
```

Comment:
It says: "In the future" in this paragraph. Why do we even have the `transRuleKey` and `transRuleArguments` *now*, if not used? It would be much better IMHO to introduce these mechanics once, we have them working in the back-end.